src/api/helpers/entityLoader.js

```diff
@@ -21,6 +21,19 @@
 
 import * as commonUtils from '../../common/helpers/utils';
 
+/**
+ * This is a middleware function to load entity detail according to given relations
+ *
+ * @param {string} modelName - Name of entity model
+ * @param {string[]} relations - List of entity relations for fetching the related detail
+ * @param {string} errMessage - Error message for responce, if any error will occur
+ * @returns {object} responce - return entity datail with status code
```

Comment:
`responce` -> `response` (Do a global search, I saw at least one other instance)
`datail` -> `detail

Reply:
`response - return entity detail with status code` does not give the full story. You should list the possible return codes, and the conditions for each, so that a user knows what to expect.